### DISCUSSION TITLE
Problems with the full node

### DISCUSSION THREAD

**Discussion by pase89** [Chia-Network/chia-blockchain#17413] [Support]
Greetings! I decided to try mining chia. Launched the graphical interface of Chia Version 2.1.4 . The wallet has synchronized. Downloaded the mainnet database from the chia website. The full node has started loading. But the problem is that the full node takes a very long time to synchronize. Synchronization from 12/21/2023 to 12/22/2023 has been going on for 3 days, which indicates to me that there is no possibility of synchronization even if I leave the computer for a year.

**Comment by BrandtH22** [comment]
Hey @pase89 , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

I would verify the peer connections and the data you are downloading from them. Generally what will occur is networks will block uPnP traffic leading to your full node only making outgoing connections and no incoming connections (you connected to others but others cannot connect to you). This leads to a limit of generally 8 peer connections rather than the default of 40-80 which can drastically increase syncing times. 

I would take a look at this guide to see if that is the issue: https://docs.chia.net/node-syncing/

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
